packages/fx-core/src/core/index.ts

```diff
@@ -966,5 +969,10 @@ export class FxCore implements Core {
   }
 }
 
+function isAzureProject(solutionSettings: SolutionSettings): boolean {
```

Comment:
This API is also useful in `solution.ts`,  to reduce duplicate code between core & solution, can we move it to some common utility?

Reply:
That would bring a lots of change in `solution.ts`, and make the target of this PR not clear . How about refactoring it in another PR?

Reply:
Sure~ Another PR will be more reasonable.